west.yml

```diff
@@ -144,6 +146,10 @@ manifest:
       repo-path: mcuboot
       path: modules/tee/tfm-mcuboot
       revision: 1.7.0-rc1
+    - name: uf2
+      repo-path: uf2
+      remote: microsoft
```

Comment:
would need a fork into Zephyr organization, as described:
https://docs.zephyrproject.org/latest/guides/modules.html#module-repositories

- All modules included in the default manifest shall be hosted in repositories under the zephyrproject-rtos GitHub organization.
- The module repository codebase shall include a module.yml file in a zephyr/ folder at the root of the repository.
- ...

Reply:
as well as this:
```suggestion
```